.travis.yml

```diff
@@ -3,6 +3,7 @@ php:
   - "7.2"
   - "7.3"
   - "7.4"
+  - "8.0"
```

Comment:
Per https://travis-ci.community/t/php-8-0-missing/10132/17, we may just need `8.0.0` not `8.0`. 

See also https://github.com/php-build/php-build/pull/651/files